tests/e2e/e2e_test.go

```diff
@@ -25,51 +25,28 @@ import (
 var t = strconv.FormatInt(time.Now().Unix(), 10)
 var projName = fmt.Sprintf("ocdev-%s", t)
 
-func runCmd(cmdS string) string {
+func runCmd(cmdS string) (string, error) {
+	cmd := exec.Command("/bin/sh", "-c", cmdS)
+	session, err := gexec.Start(cmd, GinkgoWriter, GinkgoWriter)
 
-	var cmd *exec.Cmd
-	var out, stderr bytes.Buffer
+	// wait for the command execution to complete
+	<-session.Exited
+	Expect(session.ExitCode()).To(Equal(0))
```

Comment:
would it make sense to also check `err` (`Expect(err).NotTo(HaveOccurred`) here?
So we don't have to think about checking it after every `runCmd` call.

Reply:
If we check err here, then we don't have to even return it. Or am I missing something?

Reply:
do we still need that even if `Expect` 'failed'?

Reply:
@kadel let's leave out `Fail` for now. It is not so crucial as I thought it would be. Let's come back to it when we see the need.